htc_vive.rc

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
This file is for optional file for sourcing SteamVR runtime path.
We need these additional libraries path when we use HTC VIVE.
Same descriptions are also written in the launch file so this file is optional.